.github/workflows/pull-request.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - develop
+      - commonjs
```

Comment:
Will we remove this line before the merge?

Reply:
We have not set up github workflows yet, if we plan to use workflows in the future, we will  still have to modify all the files in the .github/workflows folder, I think we can leave everything as is for now. What do you think?